src/main/java/com/fynd/extension/service/WebhookService.java

```diff
@@ -106,6 +106,9 @@ void syncSubscriberConfig(SubscriberResponse subscriberResponse, String configTy
 
             SubscriberConfigRequestV2 subscriberConfig = new SubscriberConfigRequestV2();
             if (Objects.isNull(subscriberResponse) && Objects.nonNull(this.webhookProperties)) {
+                if((configType.equals("rest") && this.restEventMap.isEmpty()) || (configType.equals("kafka") && this.kafkaEventMap.isEmpty())){
+                    return;
```

Comment:
we should give error here I believe

Reply:
In case, the user have only provided an event with provider `rest`, we should not call register API for provider `kafka` and visa versa.

So, here we can't throw error if user have provided events with only single type of provider